packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -59,12 +59,18 @@ class EksClusterStack extends TestStack {
       nodeRole: cluster.defaultCapacity ? cluster.defaultCapacity.role : undefined,
     });
 
-    // // apply a kubernetes manifest
+    // apply a kubernetes manifest
     cluster.addResource('HelloApp', ...hello.resources);
 
-    // // add two Helm charts to the cluster. This will be the Kubernetes dashboard and the Nginx Ingress Controller
+    // add two Helm charts to the cluster. This will be the Kubernetes dashboard and the Nginx Ingress Controller
     cluster.addChart('dashboard', { chart: 'kubernetes-dashboard', repository: 'https://kubernetes-charts.storage.googleapis.com' });
-    cluster.addChart('nginx-ingress', { chart: 'nginx-ingress', repository: 'https://helm.nginx.com/stable', namespace: 'kube-system' });
+    cluster.addChart('nginx-ingress', {
+      chart: 'nginx-ingress',
```

Comment:
Expecting to see an integ test update

Reply:
yes, i am update the integ test to check the logic for the both wait and timeout